src/kernels/gabor.jl

```diff
@@ -0,0 +1,74 @@
+import Base.getproperty
+
+"""
+    GaborKernel(; ell::Real=1.0, p::Real=1.0)
+
+Gabor kernel with length scale ell and period p. Given by
+```math
+    κ(x,y) =  h(x-z), h(t) = exp(-sum(t.^2./(ell.^2)))*cos(pi*sum(t./p))
+```
+
+"""
+struct GaborKernel{T<:Real, K<:Kernel} <: BaseKernel
+    κ::K
```

Comment:
Maybe use something more descriptive such as
```suggestion
    kernel::K
```
?